src/core/control/XournalMain.cpp

```diff
@@ -457,8 +457,11 @@ void on_startup(GApplication* application, XMPtr app_data) {
             opened = app_data->control->newFile("", p);
         }
     } else if (app_data->control->getSettings()->isAutoloadMostRecent()) {
-        if (auto p = Util::fromUri(gtk_recent_info_get_uri(app_data->control->getRecentManager()->getMostRecent()))) {
-            opened = app_data->control->openFile(*p);
+        auto most_recent = app_data->control->getRecentManager()->getMostRecent();
+        if (most_recent != nullptr) {
```

Comment:
Minor nitpick, but this can be simplified to
```suggestion
        if (auto* most_recent = app_data->control->getRecentManager()->getMostRecent()) {
```

Reply:
Jep, but I am not shure, if I should fix the code from other PR's in a merge.

Reply:
in general, I don't think we should. It could make things more complicated if another fix, say in 1.1.2, touches this bit of code. Then merging this other fix into master would require even more care.

In the present case, it's pretty simple and the above scare may be unjustified. Still, I think we should stick to this basic principle.